bioconda_utils/pkg_test.py

```diff
@@ -0,0 +1,104 @@
+import subprocess as sp
+import tempfile
+import tarfile
+import os
+import shlex
+
+from conda_build.metadata import MetaData
+
+'mulled-build build-and-test "gffutils" --test "python -c \'import gffutils\''
+
+
+
+
+
+def get_tests(path):
+    "Extract tests from a built package"
+    tmp = tempfile.mkdtemp()
+    t = tarfile.open(path)
+    t.extractall(tmp)
+    input_dir = os.path.join(tmp, 'info', 'recipe')
+
+    tests = []
+    recipe_meta = MetaData(input_dir)
+
```

Comment:
Ok, got it. I might look at it if I can improve it on our side but nothing urgent! Thanks.